public/static/locales/en/policy.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Created this file just for the title. It was copy pasted from the markdown file. Unless there is a way to get only the title from the markdown file this is probably the only way?